README.md

```diff
@@ -79,6 +79,33 @@ chrome_driver_name = chromedriver
 firefox_driver_name = gecodriver
 ;Browser to use in a testing (depending on this field value - appropriate driver name will be looked up in webdriver_folder)
 browser = chrome
+;Swither that allow or forbid teardown of webdriver server after test session
```

Comment:
We changed webdriver_implicit_wait_time  and webdriver_default_wait_time  to implicit_wait_time and default_wait_time

Reply:
done